plugins/periskop/package.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
like the previous comment - see if applying this role change fixes the build

https://github.com/backstage/backstage/pull/9254/files#diff-bf524eb836c5179cfb6f4c161875cdce8a980e5c36248d0f4e9814fcfdb486a8R13